.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,62 @@
+/*
+A devcontainer capable of running the local examples (in ../examples/local),
+including vtadmin-api and vtadmin-web.
+
+Note that all paths are relative to the .devcontainer directory, so ".." is the
+repository root.
+
+To run:
+1. In vscode's command palette, select "Remote-Containers: Open Folder in Container"
+2. Select the root vitess checkout from the file finder.
+3. Wait for the container to build. This can take a while.
+   ....... keep waiting
+   At this point vscode will open a new window which is the devcontainer.
+4. Accept the prompt to install go-tools (gopls, staticcheck, and so on).
+   You can also use the command palette from this window to install/update them
+   just like you would in any old vscode instance.
+5. To run the local example:
+   a. cd examples/local
+   b. ./101_initial_cluster.sh
+   c. HOST=127.0.0.1 ./scripts/vtadmin-up.sh
```

Comment:
vtadmin-up.sh is now included in the 101 script, right?

Reply:
it is on main

Reply:
maybe we can add the `HOST=127.0.0.1` variable when we execute `101_initial_cluster.sh` so we can still specify the host for vtadmin and remove the call to `vtadmin-up.sh` (after we merge main onto this branch)